AlwaysAllowChargingSounds/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="scope">
+        <item>android</item>
```

Comment:
I saw an exception in the logcat due to the class not being found, but manual testing shows the hook is indeed working. Might need to investigate further.